test/JDBC/expected/BABEL-UNSUPPORTED.out

```diff
@@ -576,6 +593,27 @@ GO
 ~~ERROR (Message: database "db_unsupported6" does not exist)~~
 
 
+CREATE DATABASE db_unsupported7 WITH CATALOG_COLLATION = DATABASE_DEFAULT
```

Comment:
This is probably not a valid test case. Please remove it.

Reply:
Its a valid test case , have verified it locally